src/main/java/blusunrize/immersiveengineering/common/blocks/metal/SampleDrillTileEntity.java

```diff
@@ -306,6 +289,7 @@ else if(!this.active)
 		return false;
 	}
 
+	/* TODO: I think these were for computer integration?
```

Comment:
Either that or for something else that was removed/became obsolete at some point. Feel free to remove them.